keycloak-config/src/main/java/org/alvearie/keycloak/config/Main.java

```diff
@@ -0,0 +1,111 @@
+/*
+(C) Copyright IBM Corp. 2021
+
+SPDX-License-Identifier: Apache-2.0
+*/
+package org.alvearie.keycloak.config;
+
+import org.alvearie.keycloak.config.util.KeycloakConfig;
+import org.alvearie.keycloak.config.util.PropertyGroup;
+import org.alvearie.keycloak.config.util.PropertyGroup.PropertyEntry;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.keycloak.admin.client.Keycloak;
+import org.keycloak.admin.client.KeycloakBuilder;
+import org.keycloak.admin.client.resource.RealmsResource;
+
+/**
+ * Reads the KeycloakConfig and applies it to the target Keycloak
+ *
+ * Command line arguments:
+ *   configFile: the file path to a JSON file containing Keycloak configuration properties
+ */
+public class Main {
+
+    private static final String CONFIG_FILE_PATH_OPTION = "configFile";
+    private static final String CLASS_NAME = "InitializeKeycloakConfig";
```

Comment:
Would be good to have the CLASS_NAME and actual class name be the same.

Reply:
```suggestion
    private static final String CLASS_NAME = "Main";
```